great_expectations/expectations/core/expect_column_values_to_be_increasing.py

```diff
@@ -97,25 +96,6 @@ class ExpectColumnValuesToBeIncreasing(ColumnMapExpectation):
     }
     args_keys = ("column",)
 
-    @public_api
-    def validate_configuration(
-        self, configuration: Optional[ExpectationConfiguration] = None
-    ) -> None:
-        """Validates the configuration of an Expectation.
-
-        The configuration will also be validated using each of the `validate_configuration` methods in its Expectation
-        superclass hierarchy.
-
-        Args:
-            configuration: An `ExpectationConfiguration` to validate. If no configuration is provided, it will be pulled
-                           from the configuration attribute of the Expectation instance.
-
-        Raises:
-            InvalidExpectationConfigurationError: The configuration does not contain the values required by the
-                                                  Expectation.
-        """
-        super().validate_configuration(configuration)
```

Comment:
This is done quite often in our codebase - no need to override if you're just going to leverage the parent impl